README.md

```diff
@@ -91,113 +94,223 @@ The front end of the integration is now complete.
-                    // Otherwise, set isProcessed at false and the Yoti SDK
-                    // will make the call.
-                    return false
-                })
-                .setBackendCompletion({ (data, error) in
-                    // when the callback has been invoked from the SDK, you can get the
-                    // data/error here. This needs to be defined even if it is empty.
-                })
-                .create()
-            
-        } catch {
-            // error management here
-            print(error)
+        guard let url = URL(string: "YOUR_CALLBACK_URL") else {
+            return false
         }
+        //Here we need to add as many scenarios as we want. each scenario is linked to a button in the Main.storyboard.
+        let firstScenario = try ScenarioBuilder().setUseCaseID("YOUR_FIRST_USE_CASE_ID")
+        .setClientSDKID("YOUR_FIRST_CLIENT_SDK_ID")
+        .setScenarioID("YOUR_FIRST_SCENARIO_ID_1")
+        .setCallbackBackendURL(url)
+        .create()
+        YotiSDK.add(scenario: firstScenario)
+                 
+        let secondScenario = try ScenarioBuilder().setUseCaseID("YOUR_SECOND_USE_CASE_ID")
+        .setClientSDKID("YOUR_SECOND_CLIENT_SDK_ID")
+        .setScenarioID("YOUR_SECOND_SCENARIO_ID_2")
+        .setCallbackBackendURL(url)
+        .create()
+        YotiSDK.add(scenario: secondScenario)
+             
+	} catch {
+		// handle error code here
+    }
+}
 ```
+Objective-C:
+Please add the scenarion method in your appDelegate.m in
 
-Objective-C
 ```objective-c
+
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+```
+
+```objective-c
+
 #import <YotiButtonSDK/YotiButtonSDK.h>
 
-NSError* error = nil;
-
-YTBScenarioBuilder *myBuilder = [[YTBScenarioBuilder alloc] init];
-
-myBuilder.useCaseID = @"YOUR_USE_CASE_ID";
-myBuilder.clientSDKID = @"YOUR_CLIENT_SDK_ID";
-myBuilder.scenarioID = @"YOUR_SCENARIO_ID";
-myBuilder.callbackBackendURL = [NSURL URLWithString:@"YOUR_CALLBACK_URL"]
-myBuilder.clientCompletion = (^BOOL(NSURL * _Nullable baseURL,
-                                    NSString * _Nullable token,
-                                    NSURL * _Nullable url,
-                                    NSError * _Nullable error) {
-  // If you decide to call the callback url via
-  // your own Service and handle the result, set isProcessed at true.
-  // Otherwise, set isProcessed at false and the Yoti SDK
-  // will make the call.
-  return isProcessed
-});
-myBuilder.backendCompletion = ((NSData * data, NSError * _Nullable error) {
-    // when the callback has been invoked from the SDK, you can get the
-    // data/error here. This needs to be defined even if it is empty.
-});
-
-YTBScenario *myScenario = [myBuilder create:&error];
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
```

Comment:
Remove that method signature + comment to match the swift documentation

Reply:
done